examples/tutorials/Introduction_to_ADCNet.ipynb

```diff
@@ -0,0 +1,1562 @@
+{
```

Comment:
<p>Let's use the version in DeepChem if possible</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepchem/deepchem/pull/4478/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='44'/>